lib/msal-common/src/logger/Logger.ts

```diff
@@ -114,17 +262,19 @@ export class Logger {
         const timestamp = new Date().toUTCString();
 
         // Add correlationId to logs if set, correlationId provided on log messages take precedence
-        const logHeader = `[${timestamp}] : [${options.correlationId}]`;
+        const logHeader = `[${timestamp}] : [${correlationId}]`;
 
         const log = `${logHeader} : ${this.packageName}@${
             this.packageVersion
         } : ${LogLevel[options.logLevel]} - ${logMessage}`;
-        // debug(`msal:${LogLevel[options.logLevel]}${options.containsPii ? "-Pii": Constants.""}${options.context ? `:${options.context}` : Constants.""}`)(logMessage);
-        this.executeCallback(
-            options.logLevel,
-            log,
-            options.containsPii || false
-        );
+
+        // Display log message only if it's not hashed
```

Comment:
Aren't we expecting all log messages to be hashed now? Shouldn't we still execute the callback with the hash?

Reply:
Oh I see now you're executing the callback in the performance client. A couple questions with that:
1. How does this work for customers not using performanceClient e.g. 3P customers?
2. Since we're grouping logs by correlationId, what mechanisms do we have to output logs in timestamp order irrespective of correlationId? e.g. if I'm debugging issues related to more than 1 request running concurrently?